packages/create-redwood-app/src/telemetry.js

```diff
@@ -9,9 +12,14 @@ import {
 import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
 import ci from 'ci-info'
 import envinfo from 'envinfo'
+import fs from 'fs-extra'
 import system from 'systeminformation'
 
-import { name as packageName, version as packageVersion } from '../package'
+// JSON modules aren't stable yet, but if they were we could use them instead.
+// See https://nodejs.org/dist/latest-v18.x/docs/api/esm.html#json-modules.
```

Comment:
This was interesting to me, so I did some digging.
It's still experimental even in Node 20: https://nodejs.org/docs/v20.0.0/api/esm.html#json-modules

It looks like it'll change syntax slightly before it fully ships (`assert` will change to `with`)
https://github.com/tc39/proposal-import-attributes
Scrolling to the very bottom there's an interesting "History" section. That section also mentions that compatibility with `assert` will probably remain even after the official syntax switches to `with`

TIL 🙂